### DISCUSSION TITLE
How to use windows absolute path for exe in loop/for/while?

### DISCUSSION THREAD

**Discussion by idlercloud** [nushell/nushell#9520] [Q&A] [answered]
`for i in 0..3 { E:/mytool/checksum/target/release/checksum.exe --version }` failed with `Error: nu::parser::parse_mismatch...expected record`. I think the path is parsed as a record because of the `:`.
Currently I solve it by using `for i in 0..3 { /mytool/checksum/target/release/checksum.exe --version }`, and it will use the current drive letter. But how can I use the whole absolute path?

**Comment by hustcer** [comment, marked as answer]
You can try 
```nushell 
^'E:/mytool/checksum/target/release/checksum.exe' --version 
```
For more detail see: https://www.nushell.sh/book/escaping.html#escaping-to-the-system

**Reply by idlercloud** [reply]
Wow, thank you for quick answer.
